framework/core/src/Post/CommentPost.php

```diff
@@ -44,9 +45,10 @@ class CommentPost extends Post
      * @param string $content
      * @param int $userId
      * @param string $ipAddress
+     * @param ServerRequestInterface|null $request
      * @return static
      */
-    public static function reply($discussionId, $content, $userId, $ipAddress)
+    public static function reply($discussionId, $content, $userId, $ipAddress, $request)
```

Comment:
Can we pass the actor instead since that's what we need? also it needs to be optional here otherwise it'll be a breaking change.